File: s3-inf-ppo/laboratoria/lab02/lab02a.java
import java.util.ArrayList;
import java.util.Random;

public class lab02a {

	static int students_count = 10;

	public static void main(String[] args) {
		ArrayList<String> students = new ArrayList<String>();
		
		for(int i = 0; i < students_count; i++) {
			students.add(getRandomStudentNumber());
		}
		
		System.out.println("Students group have been filled.");
		
		for(int i = 0; i < students.size(); i++) {
			System.out.println(students.get(i));
		}
	}

	protected static String getRandomStudentNumber() {
		Random rand = new Random();
		return String.valueOf(rand.nextInt(2000) + 38000);
	}

}

File: s3-inf-ppo/laboratoria/lab02/lab02a.cs
using System;
using System.Collections.Generic;

public class Lab02a
{
    static Random rand = new Random();
    const int students_count = 10;

    public static void Main(string[] args)
    {
        List<string> students = new List<string>();

        for (int i = 0; i < students_count; i++)
        {
            students.Add(GetRandomStudentNumber());
        }

        Console.WriteLine("Students group have been filled.");

        for (int i = 0; i < students.Count; i++)
        {
            Console.WriteLine(students[i]);
        }
        Console.ReadKey();
    }

    protected static string GetRandomStudentNumber()
    {
        return (rand.Next(2000) + 38000).ToString();
    }
}